invisible_cities/cities/penthesilea.py

```diff
@@ -3,91 +3,25 @@
-from .. cities.base_cities     import EventLoop
-from .. evm.event_model        import Cluster
-from .. evm.ic_containers      import PmapVectors
-from .. reco                   import tbl_functions as tbl
-
-from .. filters.s1s2_filter    import s1s2_filter
-from .. filters.s1s2_filter    import s2si_filter
-from .. filters.s1s2_filter    import S12Selector
 
 class Penthesilea(HitCity):
-    """Read PMAPS and produces hits and beyond"""
+    """Read PMAPS and produces a HDST"""
 
     def __init__(self, **kwds):
-        """actions:
-        1. inits base city
-        2. inits counters
-        3. inits s1s2 selector
-
-        """
         super().__init__(**kwds)
-        conf = self.conf
-
-        self.cnt.init(n_events_tot = 0,
-                      nevt_out     = 0)
-        self.drift_v        = conf.drift_v
-        self._s1s2_selector = S12Selector(**kwds)
-
-
-    def event_loop(self, pmapVectors):
-        """actions:
-        1. loops over all PMAPS
-        2. filter pmaps
-        3. write hit_event
-        """
-
-        write_hits = self.writers
-        event_numbers= pmapVectors.events
-        timestamps = pmapVectors.timestamps
-        s1_dict = pmapVectors.s1
-        s2_dict = pmapVectors.s2
-        s2si_dict = pmapVectors.s2si
-
-        for evt_number, evt_time in zip(event_numbers, timestamps):
-            self.conditional_print(self.cnt.n_events_tot, self.cnt.nevt_out)
-
-            # Count events in and break if necessary before filtering
-            what_next = self.event_range_step()
-            if what_next is EventLoop.skip_this_event: continue
-            if what_next is EventLoop.terminate_loop : break
-            self.cnt.n_events_tot += 1
-
-            # get pmaps
-            s1, s2, s2si = self. get_pmaps_from_dicts(s1_dict,
-                                                      s2_dict,
-                                                      s2si_dict,
-                                                      evt_number)
-            # filtering
-            # loop event away if any signal (s1, s2 or s2si) not present
-            if s1 == None or s2 == None or s2si == None:
-                continue
-            # filters in s12 and s2si
-            f1 = s1s2_filter(self._s1s2_selector, s1, s2, s2si)
-            f2 = s2si_filter(s2si)
-            if not f1 or not f2:
-                continue
-            # event passed selection: increment counter and write
-            self.cnt.nevt_out += 1
-            pmapVectors = PmapVectors(s1=s1, s2=s2, s2si=s2si,
-                                      events     = evt_number,
-                                      timestamps = evt_time)
-            evt = self.create_hits_event(pmapVectors)
-            write_hits(evt)
 
     def get_writers(self, h5out):
         """Get the writers needed by dorothea"""
         return  hits_writer(h5out)
+
+    def create_dst_event(self, pmapVectors):
+        """Get the writers needed by dorothea"""
```

Comment:
The docstring of `Penthesilea.create_dst_event` is "Get the writers needed by dorothea".

1. I really dislike these vacuous docstrings: if they don't say anything that isn't blatantly obvious from the name of the function, what's the point of having them?
2. If on top of being pointless, they are also *wrong* ...

Could we please drop the habit of writing utterly pointless docstrings?

I was hoping to clear out this sort of rubbish *before* this PR was accepted.